test/integration/appsource/cymbal-bank/cymbalbank_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cymbalbank
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/terraform-google-modules/enterprise-application/test/integration/testutils"
+)
+
+func TestCymbalBank(t *testing.T) {
+
+	for _, envName := range []string{
+		"development",
+		"non-production",
+		"production",
+	} {
+		envName := envName
+		t.Run(envName, func(t *testing.T) {
+			t.Parallel()
+			multitenant := tft.NewTFBlueprintTest(t, tft.WithTFDir(fmt.Sprintf("../../../../2-multitenant/envs/%s", envName)))
```

Comment:
FYI: After https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/pull/41/files can use the gcs bucket for state

Reply:
ack